python/servo/testing_commands.py

```diff
@@ -334,6 +334,15 @@ def wptrunner(self, run_file, **kwargs):
         execfile(run_file, run_globals)
         return run_globals["run_tests"](**kwargs)
 
+    @Command('update-manifest',
+             description='run test-wpt --manifest-update SKIP_TESTS to regenerate MANIFEST.json',
+             category='testing',
+             parser=create_parser_wpt)
+    def update_manifest(self, **kwargs):
+        kwargs['test_list'].append(str('SKIP_TESTS'))
```

Comment:
I was trying to keep the string types consistent and without `str('SKIP_TESTS') ` I would end up with `test_list : [u'SKIP_TESTS']` instead of ` test_list : ['SKIP_TESTS']` which is how it will look if you just ran the command `test-wpt --manifest-update SKIP_TESTS`